image-loader/conf/routes

```diff
@@ -3,6 +3,8 @@ POST    /images                     controllers.ImageLoaderController.loadImage(
 POST    /imports                    controllers.ImageLoaderController.importImage(uri: String, uploadedBy: Option[String], identifiers: Option[String], uploadTime: Option[String], filename: Option[String])
 GET     /images/project/:imageId    controllers.ImageLoaderController.projectImageBy(imageId: String)
 
+GET     /follow/:imageId            controllers.ImageLoaderController.follow(imageId: String)
```

Comment:
This additional `follow` endpoint is redundant here. Something like this might be simpler:

1. User POSTS an  image to `/images`
2. image-loader returns a URI to `/uploadStatus/:imageId`
3. Client polls this status endpoint
4. If the status is successful, the payload will tell the client where to start polling media-api
5. If the status is unsuccessful, client shows a message

As Grid's APIs follow the [argo-spec](https://github.com/argo-rest/spec), we should be able to utilise argo's `links` here.